<!-- Starts position_diff_calc node with param -->
<launch>
    <node pkg="pose_diff_calc" type="position_diff_calc" name="position_diff_calc" output="screen">
      <!-- topic name setup -->
      <param name="gps_topic_1" type="string" value="/n280/fix"/>
      <param name="gps_topic_2" type="string" value="/garmin/fix"/>
      <param name="position_diff_topic" type="string" value="/position_diff"/>
      <param name="gps_frequency_1" type="double" value="20"/>
      <param name="gps_frequency_2" type="double" value="1"/>
      <param name="position_diff_output_frequency" type="double" value="20"/>
    </node>
</launch>
